Rakefile

```diff
@@ -14,12 +16,25 @@ end
 
 namespace :design_system do
   desc "All Design System Tests"
-  task :all do
+  task all: [:report_folders] do
     puts "Running all design system tests"
     system(
       "cd ./testing && bundle exec cucumber -p reports && cd .."
     ) || raise
   end
+
+  task :report_folders do
+    puts "Creating folder structure for this test run"
+    raise "BROWSER must be set" unless ENV["BROWSER"]
+
+    base_path = "artifacts/#{ENV['BROWSER']}/#{ENV.fetch('BROWSERSTACK_CONFIGURATION_OPTIONS', 'grid')}"
```

Comment:
can we call the fallback here either "unknown" or "non-browserstack". As locally we run these rakes sometimes.

Reply:
I've added an unknown fallback for browser